server/filemanager.js

```diff
@@ -30,7 +30,7 @@ FileManager.prototype.newFile = function(id, name, path, type, source){
 			//+ ', path: ' + path + ', type: ' + type);
 
 	if(source == undefined){
-		source = '';
```

Comment:
what's the purpose of this change?

Reply:
Well, as I was reviewing it now, I think I made this change thinking that the source variable was the path to the .css file, and I changed it because of the stylelint documentation:

"_If the directory of cacheLocation does not exist, make sure you add a trailing / on *nix systems or \ on Windows. Otherwise, stylelint assumes the path to be a file._"

But I guess I misunderstood the variable, as it's the css code itself.

Reply:
gotcha, sounds like it'd be a good idea to revert this change then.